Description: Since the 1.3.x series, Fluid no longer produces (empty)
 implementations for totally empty functions in .fl files.
Author: Aaron M. Ucko <ucko@debian.org>
Bug-Debian: http://bugs.debian.org/633476
Forwarded: no
---
 src/UI/BankUI.fl     |    5 ++++-
 src/UI/OscilGenUI.fl |    6 +++++-
 src/UI/PartUI.fl     |    5 ++++-
 src/UI/PresetsUI.fl  |    6 ++++--
 4 files changed, 17 insertions(+), 5 deletions(-)

--- yoshimi.orig/src/UI/BankUI.fl
+++ yoshimi/src/UI/BankUI.fl
@@ -51,7 +51,10 @@ decl {\#include "Misc/SynthEngine.h"} {p
 class BankProcess_ {open
 } {
   Function {process(void)} {open return_type {virtual void}
-  } {}
+  } {
+    comment {This body intentionally left blank.} {in_source not_in_header
+    }
+  }
   decl {Bank *bank;} {public
   }
 }
--- yoshimi.orig/src/UI/OscilGenUI.fl
+++ yoshimi/src/UI/OscilGenUI.fl
@@ -166,7 +166,11 @@ class OscilSpectrum {open : {public Fl_B
 
 class PSlider {open : {public Fl_Slider}
 } {
-  Function {PSlider(int x,int y, int w, int h, const char *label=0):Fl_Slider(x,y,w,h,label)} {} {}
+  Function {PSlider(int x,int y, int w, int h, const char *label=0):Fl_Slider(x,y,w,h,label)} {
+  } {
+    comment {This body intentionally left blank.} {in_source not_in_header
+    }
+  }
   Function {handle(int event)} {return_type int
   } {
     code {int X=x(),Y=y(),W=w(),H=h();
--- yoshimi.orig/src/UI/PartUI.fl
+++ yoshimi/src/UI/PartUI.fl
@@ -89,7 +89,10 @@ class PartSysEffSend {open : {public Fl_
 class PartUI_ {open
 } {
   Function {showparameters(int kititem,int engine)} {open return_type virtual
-  } {}
+  } {
+    comment {This body intentionally left blank.} {in_source not_in_header
+    }
+  }
 } 
 
 class PartKitItem {: {public Fl_Group}
--- yoshimi.orig/src/UI/PresetsUI.fl
+++ yoshimi/src/UI/PresetsUI.fl
@@ -44,8 +44,10 @@ decl {\#include "Params/Presets.h"} {pub
 
 class PresetsUI_ {} {
   Function {refresh()} {return_type {virtual void}
-  } {}
-  Function {~PresetsUI_()} {} {}
+  } {
+    comment {This body intentionally left blank.} {in_source not_in_header
+    }
+  }
 } 
 
 class PresetsUI {} {
